### ISSUE TITLE
issues with forward boost of replay

### ISSUE THREAD

**Issue by luvUavz** [Jumper251/AdvancedReplay#194]
Although it's not really a problem, the plugin lacks adjustability and flexibility. I'm not a programmer, yet still tried to fix it myself, but encountered some issues, which was kind of expected... I would really like for forward and backwad boost of replay to be adjustable in the config menu, or, even better, you could make custom items addable (e.g. +-1s +-1 tick) Also I didn't find any forums I can post my ideas at, and im sorry for dumping issue tab with this post :(

**Comment by Jumper251** [comment]
I agree it would be nice to have some sort of system where you have a list of predefined actions like `jumpTo <Time>`, ` forward <Time>` and `backwards <Time>` and use them to completely customize the items in the config.

Not sure If or when that will be added but I'll keep it open as a Todo.
